strongbox-security/strongbox-authentication-registry/src/main/java/org/carlspring/strongbox/authentication/ConfigurableProviderManager.java

```diff
@@ -218,7 +219,7 @@ public void updateAuthenticationItems(AuthenticationItems items)
             Map<String, Object> properties = authenticationProvidersRegistry.getAuthenticationProperties(item.getName());
 
             properties.put("order", item.getOrder());
-            properties.put("enabled", Boolean.TRUE.equals(item.getEnabled()));
+            properties.put(Properties.ENABLED, Boolean.TRUE.equals(item.getEnabled()));
```

Comment:
@liooh could you please double check all the `Properties` usage and ensure that it used only for domain model related places?

Reply:
I just follow the informed in the task

"... All the vertices and edges property literals should be replaced with corresponding constant from org.carlspring.strongbox.db.schema.Properties ..."

what are domain model related places?

Reply:
the task description seems right, and here it's not related with vertices or edges. most relevant packages are `org.carlspring.strongbox.gremlin.adapters` and `org.carlspring.strongbox.repositories`. Maybe a bit other packages as well, just check what is related with DB entities.